Gems/Vegetation/Code/Source/Debugger/DebugComponent.cpp

```diff
@@ -972,7 +972,7 @@ void DebugComponent::DrawDebugStats(AzFramework::DebugDisplayRequests& debugDisp
 
     debugDisplay.SetColor(AZ::Color(1.0f));
     debugDisplay.Draw2dTextLabel(
-        4.0f, 16.0f, 1.5f,
+        40.0f, 22.0f, 0.7f,
```

Comment:
suggestion: It might be nice to move this to a constant (especially if we want to reuse the 0.7f too in Z (height)) - It could be a Vector3 at file scope, just a thought... 🙂

Reply:
I thought the same but I didn't want to introduce a lot of changes in this area as I'm not too familiar with it. Ideally we would have a better way to introduce a "gutter" to allow the debug text to be attached to a corner without overlapping the Viewport UI?

Reply:
Okay fair enough 👍